### DISCUSSION TITLE
Backups rotation?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#2469] [Q&A] [answered]
Hi @ganigeorgiev I know that it is not released yet but I've noticed the recent backup commits and I was wondering whether you are planning to have backups rotation or this will have to be handled manually?

Of course I can create a crontask to deletes older files but it would be nice to have it integrated in PocketBase as it would be one less thing to manage.

Thank you for your work and can't wait for the js hooks support!

**Comment by ganigeorgiev** [comment, marked as answer]
This is already implemented (_although I guess it may not be very clearly named_).

If you are testing the `develop` branch you can manage the autobackups rotation interval via the `Max @auto backups to keep` input:

![backups_screenshot](https://i.imgur.com/QqXOdCc.png)


**Keep in mind that manually generated backups are not affected and only the auto generated backups (aka. those with `@auto_` prefix) are rotated!**

**Reply by denisgurev** [reply]
Glad to hear that it is available!

I wasn't testing the develop branch, I just saw the commit in my notifications. I'll wait for the stable release 🚀.
